src/Text/Pandoc/Shared.hs

```diff
@@ -543,6 +544,27 @@ inlineListToIdentifier exts =
       | otherwise = c == '_' || c == '-' || c == '.'
     spaceToDash = T.map (\c -> if isSpace c then '-' else c)
 
+-- | Convert an 'Inline' to plaintext.
+inlineToText :: Inline -> String
```

Comment:
This looks a lot like `stringify`, but with some differences (e.g., "\n" instead of " " for LineBreak). Maybe we can unify these functions somehow?

Reply:
I have replaced it with a call to `stringify` in this PR, thanks.

Replacing FB2 `plain` with `stringify` might be a good idea too, but it is out of scope of this PR.